### ISSUE TITLE
foccus tree

### ISSUE THREAD

**Issue by siewierski2007** [jordsta95/hoi4-national-focus-maker#16]
I have a problem, that when I want to play on my focus, I get only a few instead of everybody, I can not see why, or maybe because I've made fashion National Focuses or something else?

**Comment by jordsta95** [comment]
It could be due to an error in the exported file.

If you run the game in debug mode, it will tell you what the error is

**Comment by siewierski2007** [comment]
okay only that I can not export it to a file because it shows me that a virus has been detected

**Comment by siewierski2007** [comment]
I can't export my national focus because "virus has been detected"

**Comment by jordsta95** [comment]
What browser is this on?

I have been unable to replicate this.

I believe the error message comes up because it's downloading a zip file from a site it doesn't know/trust. But I can assure you there is no viruses in here; I wouldn't put a virus in open source code (or anything for that matter)

**Comment by siewierski2007** [comment]
and should I only create a modification which in the modification tags should I take "national focus"?

**Comment by jordsta95** [comment]
Apologies, but I don't understand. 

Something may have gotten lost in translation?

**Comment by siewierski2007** [comment]
I do not know if it's the fault that I ended up with a free winrara version to get these files and put in a modification

**Comment by jordsta95** [comment]
Yes, you can use WinRar, or any other program which allows you to open zip files, in order to extract the files into your mod's folder
